### PR TITLE
Add excludeSystemPortals query param for applications GET

### PR DESCRIPTION
**Pull request by JeethJJ**
## Purpose

Adding the new query param excludeSystemPortals introduced through [1] to the spec. This is added for IS next and asgardeo. 

[1] https://github.com/wso2/identity-api-server/pull/711

### CODE REVIEW

**Comment by JeethJJ** [comment]
IS : 
IS

Asgardeo:
Asgardeo

File: en/asgardeo/docs/apis/organization-apis/restapis/application.yaml [hunk 1]
```diff
@@ -345,6 +346,17 @@ components:
         /applications?attributes=advancedConfigurations,templateId,clientId,issuer
       schema:
         type: string
+    excludeSystemPortalsQueryParam:
+      in: query
+      name: excludeSystemPortals
+      required: false
+      description: |
+        Specifies whether to include or exclude system portals in the response. 
+        Default will be treated as false if parameter is not preset in the request.
```
Comment (himeshsiriwardana) [inline en/asgardeo/docs/apis/organization-apis/restapis/application.yaml:355]:
```suggestion
        The default value is false.
```

File: en/identity-server/next/docs/apis/restapis/application.yaml [hunk 2]
```diff
@@ -3931,6 +3932,17 @@ components:
         /applications?attributes=advancedConfigurations,templateId,templateVersion,clientId,issuer
       schema:
         type: string
+    excludeSystemPortalsQueryParam:
+      in: query
+      name: excludeSystemPortals
+      required: false
+      description: |
+        Specifies whether to include or exclude system portals in the response. 
+        Default will be treated as false if parameter is not preset in the request.
```
Comment (himeshsiriwardana) [inline en/identity-server/next/docs/apis/restapis/application.yaml:3941]:
```suggestion
        The default value is false.
```

File: en/asgardeo/docs/apis/organization-apis/restapis/application.yaml [hunk 3]
```diff
@@ -345,6 +346,17 @@ components:
         /applications?attributes=advancedConfigurations,templateId,clientId,issuer
       schema:
         type: string
+    excludeSystemPortalsQueryParam:
+      in: query
+      name: excludeSystemPortals
+      required: false
+      description: |
+        Specifies whether to include or exclude system portals in the response. 
+        The default value is false.
```
Comment (himeshsiriwardana) [inline en/asgardeo/docs/apis/organization-apis/restapis/application.yaml:355]:
```suggestion
         If not provided, the default is false, meaning system portals will be excluded.
```

**Review by himeshsiriwardana — DISMISSED** [review]

File: en/asgardeo/docs/apis/organization-apis/restapis/application.yaml [hunk 4]
```diff
@@ -345,6 +346,17 @@ components:
         /applications?attributes=advancedConfigurations,templateId,clientId,issuer
       schema:
         type: string
+    excludeSystemPortalsQueryParam:
+      in: query
+      name: excludeSystemPortals
+      required: false
+      description: |
+        Specifies whether to include or exclude system portals in the response.
```
Comment (himeshsiriwardana) [inline en/asgardeo/docs/apis/organization-apis/restapis/application.yaml:354]:
```suggestion
        Specifies whether to exclude system portals in the response. 
```

File: en/asgardeo/docs/apis/organization-apis/restapis/application.yaml [hunk 5]
```diff
@@ -345,6 +346,17 @@ components:
         /applications?attributes=advancedConfigurations,templateId,clientId,issuer
       schema:
         type: string
+    excludeSystemPortalsQueryParam:
+      in: query
+      name: excludeSystemPortals
+      required: false
+      description: |
+        Specifies whether to include or exclude system portals in the response. 
+         If not provided, the default is false, meaning system portals will be excluded.
```
Comment (himeshsiriwardana) [inline en/asgardeo/docs/apis/organization-apis/restapis/application.yaml:355]:
```suggestion
         If not provided, the default is false, meaning system portals will be included.
```

File: en/identity-server/next/docs/apis/restapis/application.yaml [hunk 6]
```diff
@@ -3931,6 +3932,17 @@ components:
         /applications?attributes=advancedConfigurations,templateId,templateVersion,clientId,issuer
       schema:
         type: string
+    excludeSystemPortalsQueryParam:
+      in: query
+      name: excludeSystemPortals
+      required: false
+      description: |
+        Specifies whether to include or exclude system portals in the response.
```
Comment (himeshsiriwardana) [inline en/identity-server/next/docs/apis/restapis/application.yaml:3940]:
```suggestion
        Specifies whether to exclude system portals in the response. 
```

File: en/identity-server/next/docs/apis/restapis/application.yaml [hunk 7]
```diff
@@ -3931,6 +3932,17 @@ components:
         /applications?attributes=advancedConfigurations,templateId,templateVersion,clientId,issuer
       schema:
         type: string
+    excludeSystemPortalsQueryParam:
+      in: query
+      name: excludeSystemPortals
+      required: false
+      description: |
+        Specifies whether to include or exclude system portals in the response. 
+        If not provided, the default is false, meaning system portals will be excluded.
```
Comment (himeshsiriwardana) [inline en/identity-server/next/docs/apis/restapis/application.yaml:3941]:
```suggestion
        If not provided, the default is false, meaning system portals will be included.
```

**Review by himeshsiriwardana — APPROVED** [review]

**Pull request merged**